Content.Server/Radiation/Events/OnRadiationSystemUpdated.cs

```diff
@@ -0,0 +1,6 @@
+namespace Content.Server.Radiation.Events;
+
+public sealed class OnRadiationSystemUpdated : EntityEventArgs
+{
```

Comment:
```suggestion
public sealed class RadiationSystemUpdatedEvent : EntityEventArgs
{
```

Update other usages.